src/serializers/Serializer.js

```diff
@@ -12,4 +24,14 @@ const classToPlain = function (object, convertCase = false) {
     return convertCase ? snakeCaseKeys(camelCasePlain) : camelCasePlain;
 };
 
-export { classToPlain, plainToClass };
+const filterImmutableFields = function (object) {
+    const filtered = cloneDeep(object);
+
+    IMMUTABLE_FIELDS.forEach(field => {
+        delete filtered[field];
+    });
+
+    return filtered;
+}
+
+export { classToPlain, plainToClass, filterImmutableFields };
```

Comment:
above export is: { classToPlain, filterImmutableFields, plainToClass }
Is here filterImmutableFields in correct position?

Reply:
Position does not matter functionally, unless you mean alphabetical order for style?

Reply:
I thought it was typo. Approved